tabs/pid_tuning.css

```diff
@@ -287,7 +302,7 @@
 }
 
 .tab-pid_tuning table td {
-    padding: 1px;
+    padding: 5px;
```

Comment:
> What platform are you testing with?

using windows 10 laptop with latest chrome

> Also, why do you attribute the problems you are getting to the change to

simply because ... the problem occurs when I change this line 305